packages/ignite/src/plugins/default/commands/new.js

```diff
@@ -29,35 +29,51 @@ module.exports = async function (context) {
     return
   }
 
+  // Install the ignite-* packages from the local dev directory.
+  //
+  // This is pretty much always what we while we dev.
+  //
+  // To test what live is like, you can run `ignite new FooTown --live`.
+  //
+  // TODO(steve): Don't forget to remove this when we launch... open to better ways of handling it.
+  const igniteDevPackagePrefix = parameters.options.live || `${__dirname}/../../../../../ignite-`
+
   // First we ask!
   const answers = await context.prompt.ask(installWalkthrough)
   // then we kick off (TODO: Would be awesome to have this kick off during questions)
   // we need to lock the RN version here
   info('Creating new RN project')
   // TODO make sure `react-antive --version has react-native-cli 2.x otherwise failure`
-  await system.run(`react-native init ${projectName} --version 0.38.0`)
+  const reactNativeVersion = '0.40.0'
```

Comment:
Thoughts on storing this in package.json?  I have not idea who/what would also care about this, but centralizing versions feels right to me.  *shrug*